README.md

```diff
@@ -0,0 +1,179 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>Portfolio: setup and mobile first</b></h3>
+</div>
+# :green_book: Table of Contents
+
+- [:book: About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [:rocket: Live Demo](#live-demo)
+- [:computer: Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [:bust_in_silhouette: Author](#author)
+- [:telescope: Future Features](#future-features)
+- [:handshake: Contributing](#contributing)
+- [:star:️ Show your support](#support)
+- [:pray: Acknowledgements](#acknowledgements)
+- [:memo: License](#license)
+
+# :book: Portfolio: setup and mobile first <a name="about-project"></a>
+
+**Portfolio: setup and mobile first** is a simple project based on HTML, CSS and Linters.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://css-tricks.com/snippets/css/a-guide-to-flexbox/">FlexBox</a></li>
+    <li><a href="https://mozilla.github.io/addons-linter/">Linters</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **HTML.**
+- **CSS.**
+- **Linters.**
+- **Flexbox.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
```

Comment:
- [optional] please remove the live demo section since your project is yet to be deployed. you can add this later on when your deployed link is available. nice job